### PR TITLE
273 stand tidyup

### PR DESCRIPTION
**Pull request by AndyTWF**

### CODE REVIEW

File: app/Services/StandService.php [hunk 1]
```diff
@@ -137,4 +139,55 @@ private function standExists(int $standId): bool
...
+     *
+     * @param string $airfield
+     * @param string $identifier
+     * @throws \Exception
+     */
+    public function deleteStand(string $airfield, string $identifier)
+    {
+        $stand = Stand::with('airfield')->whereHas(
+            'airfield',
+            function (Builder $query) use ($airfield) {
```
Comment (AxonC) [inline app/Services/StandService.php:154]: 7.4 I can't wait for you

File: app/Services/StandService.php [hunk 2]
```diff
@@ -137,4 +139,55 @@ private function standExists(int $standId): bool
...
+                $query->where('code', $airfield);
+            }
+        )->where('identifier', $identifier)
+            ->first();
+
+        if (!$stand) {
```
Comment (AxonC) [inline app/Services/StandService.php:160]: Maybe a candidate for a find or fail?

File: app/Services/StandService.php [hunk 3]
```diff
@@ -137,4 +139,55 @@ private function standExists(int $standId): bool
...
     }
+
+    /**
+     * Delete a given stand
+     *
+     * @param string $airfield
+     * @param string $identifier
+     * @throws \Exception
+     */
+    public function deleteStand(string $airfield, string $identifier)
```
Comment (AxonC) [inline app/Services/StandService.php:150]: typehint on the return type

File: app/Services/StandService.php [hunk 4]
```diff
@@ -137,4 +139,55 @@ private function standExists(int $standId): bool
...
+    /**
+     * Change the identifier for a stand
+     *
+     * @param string $airfield
+     * @param string $oldIdentifier
+     * @param string $newIdentifier
+     */
+    public function changeStandIdentifier(string $airfield, string $oldIdentifier, string $newIdentifier)
+    {
+        $stand = Stand::with('airfield')->whereHas(
```
Comment (AxonC) [inline app/Services/StandService.php:177]: is this code identical to find the identifiers? Maybe a candidate for a private function

File: app/Services/StandService.php (hunk 2 above)
Comment (AndyTWF) [reply app/Services/StandService.php:160]: Was toying with that - decided against it as this'll be mainly used in migrations and we probably wouldn't want to crash the entire migration over something not being deleted when it never existed anyway

File: app/Services/StandService.php (hunk 1 above)
Comment (AndyTWF) [reply app/Services/StandService.php:154]: Plz come quicker

File: app/Services/StandService.php (hunk 3 above)
Comment (AndyTWF) [reply app/Services/StandService.php:150]: Added

File: app/Services/StandService.php (hunk 4 above)
Comment (AndyTWF) [reply app/Services/StandService.php]: Done!

**Review by AxonC — APPROVED** [review]

**Pull request merged**